Automattic/wp-calypso
# Signup: Remove eCommerce benchmarking AB test

Discused in p1v4f2-1TI-p2.

Removes partner store option that were introduced with eCommerce benchmarking AB test, and leaves Pressable as the only option. These changes were introduced in https://github.com/Automattic/wp-calypso/pull/7162.

# Testing instructions #
1. To reach this step you will need to start in incognito window, because it's gated behind `signupStore` test, which is only available for new users. 
2. Navigate to `/start`.
3. Select `Start an online store` option.
4. Verify that Pressable store step works correctly.
5. Verify there are no JS errors in the console.
6. Verify that all remaining references to Bluehost and SiteGround store step have been cleaned up.
